misp/src/misp.py

```diff
@@ -488,12 +488,14 @@ def process_events(self, work_id, events):
                 bundle_objects.append(object_relationship)
 
             # Create the report if needed
-            if self.misp_create_report and len(object_refs) > 0:
```

Comment:
A STIX 2.1 report must have at least one object, here the STIX 2.1 library will raise an error if object_refs is empty.